pkg/azurefile/azurefile.go

```diff
@@ -54,6 +61,7 @@ func NewDriver(nodeID string) *Driver {
 	driver.Name = DriverName
 	driver.Version = driverVersion
 	driver.NodeID = nodeID
+	driver.volumes = make(map[string]*volume)
```

Comment:
store all volumes in memory is not a good solution, what happen if this driver crash or reschedule to other node